tests/schema_test_files/system/test_pass_links.json

```
@@ -0,0 +1,29 @@
+{
+  "describedBy": "https://schema.dev.data.humancellatlas.org/system/9.9.9/links.json",
+  "schema_type": "link_bundle",
```

@hannes-ucsc would Azul be affected if the schema_type changed?

---

Isn't that the same value for `schema_type` that we use in v1?

---

Yes. But there's an opportunity here to rename this to reflect the move away from bundles

---

Neither Azul nor metadata-api look at this property. We infer the entity type from the schema file name in the `describedBy` URL. That's probably the wrong thing to do but we did it because some ancient version of the schema didn't have schema_type IIRC. So this helps us now since we can change it without breaking Azul. I'd go with

```suggestion
  "schema_type": "links",
```